pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```
@@ -29,20 +26,19 @@
 , ...
 } @ args:
 
-assert (pname != null || version != null) -> (name == null && pname != null); # You must declare either a name or pname + version (preferred).
+assert (!args ? pname || !args ? version) -> (args ? name); # You must provide name if pname or version (preferred) is missing.
 
 with builtins;
 let
-  pname = if args ? name && args.name != null then args.name else args.pname;
-  versionStr = lib.optionalString (version != null) ("-" + version);
-  name = pname + versionStr;
+  name = args.name or "${args.pname}-${args.version}";
+  executableName = args.pname or args.name;
+  nameAttrs = lib.filterAttrs (key: value: builtins.elem key ["name" "pname" "version"]) args;
```

```suggestion
  nameAttrs = lib.filterAttrs (key: value: builtins.elem key [ "name" "pname" "version" ]) args;
```

This is a really odd pattern but I don't see an obvious way of doing it better at first glance either.

Perhaps just simply always pass name to runCommandLocal or, even simpler, don't pass it at all. It's intended to be called with a simple name anyways I think.